Libraries/LibWeb/HTML/EventLoop/Task.h

```diff
@@ -33,6 +33,8 @@ class Task final : public JS::Cell {
         PostedMessage,
         Microtask,
         TimerTask,
+        // https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#bitmap-task-source
+        BitmapTask,
```

Comment:
Let's put this a bit lower so we don't accidentally assume `JavascriptEngine` is part of this spec.